Drivers/Src/BatteryBalancing.c

```diff
@@ -0,0 +1,78 @@
+
+#include "LTC6811.h"
+#include "Voltage.h"
+#include "BatteryBalancing.h"
+#include "config.h"
+#include "Current.h"
+
+#define CHARGING_TOLERANCE 0
+
+static uint16_t Balancing_Init(void) {
+	uint16_t minVoltage = 0;
+	if (Current_CheckStatus(0) == 1){
+		return;//If charging, return
+	}
+	Voltage_UpdateMeasurements();
```

Comment:
Why are we doing this in here? We should be using the most recent voltage measurements (they are stored in a buffer by Voltage.c. I think it would make more sense to either:
1. Add a function to Voltage.c to find the minimum voltage of all of the modules
2. Add a function to Voltage.c to access the buffer of stored voltage values (`VoltageVal`)
3. Use  `Voltage_GetModuleMillivoltage()` without calling `Voltage_UpdateMeasurements()` first
I am also open to other ways to do this, if you have a better idea